docs/reference/objects/cart/index.md

```diff
@@ -27,10 +27,10 @@ The currency code associated with the cart.
 
 ## `cart.custom_data`
 {: .d-inline-block }
-object
+array [Custom Data]({% link docs/reference/objects/cart/custom_data.md %})
 {: .label .fs-1 }
 
-An object containing any custom data properties associated with the cart
+An array containing [Custom Data]({% link docs/reference/objects/cart/custom_data.md %}) of booking custom fields with values.
```

Comment:
Can we please lower case custom data here please? Again just because it seems to be the preferred format throughout the docs

Reply:
Should we mention that these are only going to return the exposed ones?